src/main/resources/META-INF/rust-core.xml

```diff
@@ -1273,6 +1273,8 @@
             default="1000"
             groupKey="advanced.setting.rust.group"
         />
+
+        <iconMapper mappingFile="RustIconMappings.json"/>
```

Comment:
nit: let's move it to other icon-related extensions. For example, to `<!-- Icon Provider -->` section and rename section to `Icons`